composer.json

```diff
@@ -16,14 +16,15 @@
     ],
     "require": {
         "php": "^7.1.3",
-        "illuminate/contracts": "5.7.*",
-        "illuminate/support": "5.7.*",
+        "illuminate/contracts": "^5.8",
+        "illuminate/support": "^5.8",
         "graham-campbell/manager": "^4.0",
         "vimeo/vimeo-api": "^3.0"
     },
     "require-dev": {
+        "laravel/framework": "^5.8",
```

Comment:
Without explicitly requesting 5.8 here, `graham-campbell/testbench` was installing `laravel/framework` 5.7, which would then cause problems for other dependencies. Really strange.